tests/utils/next-version-helpers.mjs

```diff
@@ -97,8 +96,17 @@ export async function setNextVersionInFixture(
           return
         }
         packageJson.dependencies.next = version
+
+        const { stdout } = await execaCommand(
+          `npm info next@${resolvedVersion} peerDependencies --json`,
+          { cwd },
+        )
+
+        const nextPeerDependencies = JSON.parse(stdout)
```

Comment:
`next@canary` started to bump react(-dom) peer dependencies quite frequently, so instead of hardcoding react version - we just check what react version given next version needs